src/pages/calculator.tsx

```diff
@@ -0,0 +1,386 @@
+import FinalAmount from '@components/calculator/FinalAmount';
+import CTA from '@components/CTA';
+import {
+	FormInput,
+	FormInputWrapper,
+	FormRadioGroup,
+	FormRadioLabel,
+	FormRadioWrapper,
+	FormSelect,
+} from '@components/styled/Forms';
+import Link from 'next/link';
+import { useState } from 'react';
+import styled from 'styled-components';
+import {
+	calculateCylinderResinInKg,
+	calculateRectangularResinInKg,
+} from 'utils/calculator';
+
+export enum PRODUCTTYPE {
+	OEX5302 = 'OEX5302',
+	OEX5311 = 'OEX5311',
+}
+export enum SHAPETYPE {
```

Comment:
You exported this but it's not used anywhere else

Reply:
moved it to the utils file